karaf/deps-brooklyn/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+        <artifactId>brooklyn-tosca-karaf</artifactId>
+        <version>0.10.0-SNAPSHOT</version> <!-- BROOKLYN_TOSCA_VERSION -->
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>brooklyn-tosca-karaf-deps-brooklyn</artifactId>
+    <packaging>jar</packaging>
+    <name>Brooklyn TOSCA Karaf OSGI Brooklyn Dependencies</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-camp</artifactId>
+            <version>${project.version}</version>
```

Comment:
This should be `brooklyn.version`. We don't want to rely upon `BROOKLYN_TOSCA_VERSION` matching exactly the brooklyn version.